### DISCUSSION TITLE
GitHub staff is unresponsive here

### DISCUSSION THREAD

**Discussion by falsandtru** [community/community#37572] [Other Feature Feedback, Questions, & Ideas]
GitHub staff does not seem to respond to us here. This place seems to be a place to let people vent prepared by GitHub. Our votes for the statement of opposition would not even be accurately counted because **uniquely counting all votes to all statements of opposition requires tedious additional work**. **Therefore, it is doubtful that the feedback here is accurately statisticalized.** To not keep showing garbage feeds, we need to take definitely effective steps. Let's first individually [tell](https://support.github.com/contact/account) the staff of our opposition and have them count it to statisticalize our votes. And **we need to get GitHub to answer how GitHub is accurately statisticalizing our feedback**. Are there any other effective methods?

**Comment by falsandtru** [comment]
GitLab will be greatly pleased to hear of this deterioration.

**Comment by falsandtru** [comment]
Here is the answer from GitHub support. I wish everyone who sees this will be satisfied.

> GitHub Support commented
Hi falsandtru,
>
>Thanks for contacting GitHub Support! 😊
>
>I can certainly clear the following feed for you. This will clear all public and private events from your "Following" feed completely. Would you like me to continue?
>
>I am looking forward to hearing back from you 😊
>
>Best wishes,
Pedro
>
>🌯 Find answers to common questions and learn with other GitHub users in our new [GitHub Community Forum](https://github.community/) 🌯

**Comment by hmmhmmhm** [comment]
Why not provide a button to return to its original state while conducting this a/b test? I'm very embarrassed that I have to request by e-mail to go back.

**Comment by falsandtru** [comment]
Has anyone else inquired through the contact form?

**Comment by DmitriiShamrikov** [comment]
One more reason to move permamently to GitLab
